.env.test

```diff
@@ -0,0 +1,71 @@
+# .env.test
```

Comment:
idk why this file is listed in my diff when I just merged off of develop...

Reply:
if this file was unintentionally committed, I'd recommend removing this from this PR, either by removing it, or adding .env.test to .gitignore. This file doesn't exist in develop so it'll be created upon merge.